crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -0,0 +1,131 @@
+//! Run a prepass before the main pass to get the depth and/or normals texture.
+//! The depth prepass texture is then used by the main pass to reduce overdraw.
+//!
+//! To enable the prepass, you need to add a `PrepassSettings` component to a `Camera`.
+//! Both textures are available on the `PrepassTextures` component attached to each `Camera` with a `PrepassSettings`
+//!
+//! Currently only works for 3d
+
+pub mod node;
+
+use bevy_ecs::prelude::*;
+use bevy_render::{
+    render_phase::{CachedRenderPipelinePhaseItem, DrawFunctionId, EntityPhaseItem, PhaseItem},
+    render_resource::{CachedRenderPipelineId, Extent3d},
+    texture::CachedTexture,
+};
+use bevy_utils::FloatOrd;
+
+/// Add a `PrepassSettings` component to a `Camera` to perform a depth and/or normal prepass.
+/// These textures are useful for various screen-space effects and reducing overdraw in the main pass.
+///
+/// The textures are automatically added to the default mesh view bindings. You can also get the raw textures
+/// by querying the `ViewPrepassTextures` component on the camera with the `PrepassSettings`.
+#[derive(Clone, Component)]
+pub struct PrepassSettings {
+    /// If true then depth values will be copied to a separate texture available to the main pass.
+    /// This will help reduce overdraw in the main pass.
+    pub output_depth: bool,
```

Comment:
Is it feasible to change `output` to a synonym? As a relative beginner, this is challenging to read, as I interpret it as "the depth of the output", not "should pass through depth/normal values".

Reply:
The way I read it is "Should the prepass output the depth", but I see what you mean, I'll try to come up with something better.